link-parser/parser-utilities.h

```diff
@@ -0,0 +1,57 @@
+/***************************************************************************/
+/* Copyright (c) 2016 Linas Vepstas                                        */
```

Comment:
Uh, Could be your name here, too. I see that its mostly a cut-n-paste but really you're doing the work.